src/main/java/core/basesyntax/SalaryInfo.java

```diff
@@ -1,7 +1,36 @@
 package core.basesyntax;
 
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+
 public class SalaryInfo {
+    private static final int INDEX_OF_NAME_ZERO = 0;
+    private static final int INDEX_OF_NAME_FIRST = 1;
+    private static final int INDEX_OF_NAME_SECOND = 2;
+    private static final int INDEX_OF_NAME_THIRD = 3;
```

Comment:
No, something like this, that we can read and understand what data we get from this index:
INDEX_OF_DATE, INDEX_OF_NAME, INDEX_OF_HOUR